nomt/src/store/writeout.rs

```diff
@@ -404,15 +384,16 @@ impl MetaSwap {
 
             if let Err(_) = io.try_send_meta(IoKind::Write(self.meta_fd, 0, page)) {
                 self.new_meta = Some(new_meta);
-                return false;
             }
         }
 
         if self.should_fsync {
-            if let Err(_) = io.try_send_meta(IoKind::Fsync(self.meta_fd)) {
-                return false;
+            unsafe {
+                let f = File::from_raw_fd(self.meta_fd);
+                f.sync_all().unwrap();
```

Comment:
it's cleaner to just call libc::fsync here

Reply:
Fair! thanks! I will update it right now!